bbmri/docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
     container_name: bridgehead-focus
     environment:
       API_KEY: ${FOCUS_BEAM_SECRET_SHORT}
-      BEAM_APP_ID: focus
+      BEAM_APP_ID: focus.${PROXY_ID}
```

Comment:
Please name either `BEAM_APP_ID_SHORT` or `BEAM_APP_ID_LONG` to avoid confusion

Reply:
done